### PR TITLE
[KO] Add `Community/osu! Discord Server`

### PR DESCRIPTION
**Pull request by Sitoria**
i tried
size:M i think

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Sitoria** [comment]
Stop fixing md file until 
https://github.com/ppy/osu-wiki/pull/9176
^ This PR is over

huge thanks for check Sylphi :))

**Comment by Sitoria** [comment]
will work
maybe via 2-3 days

File: wiki/Community/osu!_Discord_server/ko.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+tags:
+  - development
+  - osu!dev
+---
+
+# osu! 디스코드 서버
+
+**osu!dev**로도 알려져 있었던 **osu!** 디스코드 서버는 osu!의 개발 프로젝트들의 작업을 위한 공식 [디스코드](https://discordapp.com) 서버입니다. 이 서버는 osu!의 오픈소스 및 커뮤니티에 대해 토론하고 작업할 수 있는 허브 역할을 합니다. 또한 osu! 개발에 기여하고 [osu!팀](/wiki/People/osu!_team)에게 연락할 수 있는 주요 장소입니다.
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/Community/osu!_Discord_server/ko.md:9]: apply

File: wiki/Community/osu!_Discord_server/ko.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+서버의 공식 초대 링크는 <https://discord.gg/ppy>입니다.
+
+서버에 처음 오신 분들은 `#introductions` 채널에서 자신을 소개하는 글을 작성할 수 있습니다만, 서버의 다른 사람들과 교류하기 위해 반드시 필요한 것은 아닙니다.
+
+## 체널
+
+osu! 디스코드 서버는 여러 채널이 있으며, 대부분 특정 프로젝트의 개발을 주제로 사용되고 있습니다.
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/Community/osu!_Discord_server/ko.md:17]: `대부분 특정 프로젝트나 두 가지 주요 개발을 주제로 하고 있습니다` fixed

File: wiki/Community/osu!_Discord_server/ko.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+| `#difficulty-osu` | [osu!](/wiki/Game_mode/osu!) [스타 레이팅](/wiki/Beatmap/Star_rating)과 [퍼포먼스 포인트](/wiki/Performance_points) 계산 |
+| `#difficulty-taiko` | [osu!taiko](/wiki/Game_mode/osu!taiko) 스타 레이팅과 퍼포먼스 포인트 계산 |
+| `#difficulty-catch` | [osu!catch](/wiki/Game_mode/osu!catch) 스타 레이팅과 퍼포먼스 포인트 계산 |
+| `#difficulty-mania` | [osu!mania](/wiki/Game_mode/osu!mania) 스타 레이팅과 퍼포먼스 포인트 계산 |
+
+### 커뮤니티
+
+| 채널 | 설명 |
+| :-- | :-- |
+| `#featured-artists` | [피쳐드 아티스트](/wiki/People/Featured_Artists) 비트맵 제작, 프로젝트, 플레이리스트 |
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/Community/osu!_Discord_server/ko.md:50]: thanks for that i think hardly about this

File: wiki/Community/osu!_Discord_server/ko.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+| `#mappers-guild` | The [Mappers' Guild](/wiki/Community/Mappers_Guild) |
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/Community/osu!_Discord_server/ko.md:51]: erased

File: wiki/Community/osu!_Discord_server/ko.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+| `#osu-spotlights` | [비트맵 스포트라이트](/wiki/Beatmap_Spotlights) |
+| `#osu-wiki` | [osu! 위키](/wiki/osu!_wiki) |
+| `#osu-news` | [osu!월간지](/wiki/Community/osu!monthly)와 일반 뉴스 보도 |
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/Community/osu!_Discord_server/ko.md:54]: ok ._.

File: wiki/Community/osu!_Discord_server/ko.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+| `#osu-loved` | [Project Loved](/wiki/Community/Project_Loved) |
+| `#tournaments` | [토너먼트](/wiki/Tournaments)와 [공식 토너먼트 지원](/wiki/Tournaments/Official_support) |
+| `#community-meetings` | [osu! 커뮤니티 미팅](/wiki/Community/osu!_community_meetings) |
+
+### 보관됨
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/Community/osu!_Discord_server/ko.md:59]: applied

**Comment by Sitoria** [comment]
ah i got wrong commit grammar word jesus X(

**Review by TicClick — APPROVED** [review]

**Pull request merged**